.ci/jenkins/utils/utils.groovy

```diff
@@ -17,6 +17,14 @@
 STAGES_STATUS_MAP = [:]
 
 def GetDockerEnvList(String projectName, String dockerContainerNamePrefix, String projectRoot = projectName) {
+    /**
+    * Generates map for method using configurationMaps.
```

Comment:
This method generates configuration map list using dockerfiles available in dockerfiles directory